ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -140,6 +140,20 @@ if pytorch_support_is_available
 
   nnstreamer_filter_torch_deps = pytorch_support_deps + [glib_dep, gst_dep, nnstreamer_dep]
 
+  pytorch_compile_args = []
+
+  pg_pkgconfig = find_program('pkg-config')
+  cmd = run_command(pg_pkgconfig, ['--print-errors', '--exists', 'pytorch >= 1.2.0'])
+  if cmd.returncode() == 0
+    pytorch_compile_args += '-DPYTORCH_VER_ATLEAST_1_2_0=1'
+  endif
```

Comment:
Can you check pytorch version directly with this: https://mesonbuild.com/Dependencies.html#dependencies-using-config-tools ??

Reply:
Updated.